specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Yes, I think it should.  Note the commented out specification of ballot because of our lack of information about Dominion schemas when I wrote that module.